src/app/tests/suites/commands/interaction_model/InteractionModel.cpp

```diff
@@ -0,0 +1,174 @@
+/*
+ *   Copyright (c) 2022 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#include "InteractionModel.h"
+
+CHIP_ERROR InteractionModel::ReadAttribute(const char * identity, chip::EndpointId endpointId, chip::ClusterId clusterId,
+                                           chip::AttributeId attributeId, bool fabricFiltered)
+{
+    chip::DeviceProxy * device = GetDevice(identity);
+    VerifyOrReturnError(device != nullptr, CHIP_ERROR_INCORRECT_STATE);
+
+    chip::app::AttributePathParams attributePathParams[1];
+    attributePathParams[0].mEndpointId  = endpointId;
+    attributePathParams[0].mClusterId   = clusterId;
+    attributePathParams[0].mAttributeId = attributeId;
+
+    chip::app::ReadPrepareParams params(device->GetSecureSession().Value());
+    params.mpEventPathParamsList        = nullptr;
+    params.mEventPathParamsListSize     = 0;
+    params.mpAttributePathParamsList    = attributePathParams;
+    params.mAttributePathParamsListSize = 1;
+    params.mIsFabricFiltered            = fabricFiltered;
+
+    mReadClient =
+        std::make_unique<chip::app::ReadClient>(chip::app::InteractionModelEngine::GetInstance(), device->GetExchangeManager(),
```

Comment:
Is there a reason to not just `using namespace chip` and `using namespace chip::app` at the top of this file?